meta-resin-morty/classes/image_types_resin.bbclass

```diff
@@ -0,0 +1,297 @@
+inherit image_types
```

Comment:
Checking now the code I see that `IMGDEPLOYDIR` was defined as a new variable in 619d2996fbb12e6ab63daa73b2d9683023cc2744 . So everything before morty didn't have this variable defined. So I would suggest something like: check if `IMGDEPLOYDIR` is defined. If yes use it, otherwise use `DEPLOY_DIR_IMAGE`. Plus a comment that this is legacy code to support pre-morty releases.